### PR TITLE
Add vesting tests

### PR DESCRIPTION
**Pull request by tensojka**

### CODE REVIEW

**Comment by tensojka** [comment]
Closes #48 

Please review @DaveVodrazka

File: src/oz/access/ownable.cairo [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        }
+
+        /// Transfers ownership of the contract to a new address.
+        ///
+        /// Internal function without access restriction.
+        fn _transfer_ownership(
+            ref self: ComponentState<TContractState>, new_owner: ContractAddress
+        ) {
+            let previous_owner: ContractAddress = self.Ownable_owner.read();
+            self.Ownable_owner.write(new_owner);
```
Comment (DaveVodrazka) [inline src/oz/access/ownable.cairo:108]: Shouldn't `assert_only_owner();` be called somewhere in here?

File: src/vesting.cairo [hunk 2]
```diff
@@ -65,10 +71,11 @@ mod vesting {
             vested_timestamp: u64
```
Comment (DaveVodrazka) [inline src/vesting.cairo:71]: What exactly is `vested_timestamp`?

File: src/vesting.cairo [hunk 3]
```diff
@@ -65,10 +71,11 @@ mod vesting {
...
         ) {
             let amt_to_vest = self.milestone.read((vested_timestamp, grantee));
-            assert(amt_to_vest != 0, 'no vesting milestone found, or already vested');
+            assert(amt_to_vest != 0, 'nothing to vest');
             assert(get_block_timestamp() > vested_timestamp, 'not yet eligible');
-            IGovernanceTokenDispatcher { contract_address: govtoken_addr }
-                .mint(claimee, u256 { high: 0, low: amt_to_vest });
+            let state = Governance::unsafe_new_contract_state();
+            IGovernanceTokenDispatcher { contract_address: state.get_governance_token_address() }
+                .mint(grantee, u256 { high: 0, low: amt_to_vest });
```
Comment (DaveVodrazka) [inline src/vesting.cairo:78]: Is it not possible to cast u128 into u256?
```suggestion
                .mint(grantee, amt_to_vest.into());
```
Or maybe even change the storage var to `milestone: LegacyMap::<(u64, ContractAddress), u256>` to keep it consistent?

File: src/oz/access/ownable.cairo (hunk 1 above)
Comment (tensojka) [reply src/oz/access/ownable.cairo:108]: this is OpenZeppelin code. I completely removed it from the repo, it is now only a dependency.

File: src/vesting.cairo (hunk 2 above)
Comment (tensojka) [reply src/vesting.cairo:71]: Time when a vesting milestone is eligible, i.e. can be vested. In this case, it is in the past, which is why I call it vested_timestamp not vesting_timestamp

File: src/vesting.cairo (hunk 3 above)
Comment (tensojka) [reply src/vesting.cairo]: Done, no reason to change the storage var imo

**Pull request merged**